tutorials/dataframe/df106_HiggsToFourLeptons.C

```diff
@@ -0,0 +1,281 @@
+         // Create new columns with the kinematics of good leptons
+         .Define("goodlep_pt", "lep_pt[good_lep]")
+         .Define("goodlep_eta", "lep_eta[good_lep]")
+         .Define("goodlep_phi", "lep_phi[good_lep]")
+         .Define("goodlep_E", "lep_E[good_lep]")
+         .Define("goodlep_type", "lep_type[good_lep]")
+         // Select leptons with high transverse momentum
+         .Filter("goodlep_pt[0] > 25000 && goodlep_pt[1] > 15000 && goodlep_pt[2] > 10000")
+         // Compute invariant mass
+         .Define("m4l", "ComputeInvariantMass(goodlep_pt, goodlep_eta, goodlep_phi, goodlep_E)")
+         // Reweighting of the samples is different for "data" and "MC"
+         .DefinePerSample("reweighting",
+                          [](unsigned int slot, const ROOT::RDF::RSampleInfo &id) { return id.Contains("mc"); });
+
+   // Define the weight column (scale factor) for the MC samples
+   auto df_mc = df_analysis.Filter("reweighting == true")
+                   .Define("weight", ("scaleFactor_ELE * scaleFactor_MUON * scaleFactor_LepTRIGGER * "
+                                      "scaleFactor_PILEUP * mcWeight * scale * xsecs / sumws * lumi"));
+
+   // Book histograms for individual MC samples
+   auto df_higgs = df_mc.Filter("sample_category == \"higgs\"")
+                      .Histo1D<float>(ROOT::RDF::TH1DModel("higgs", "m4l", 24, 80, 170), "m4l", "weight");
+   auto df_zz = df_mc.Filter("sample_category == \"zz\"")
+                   .Histo1D<float>(ROOT::RDF::TH1DModel("zz", "m4l", 24, 80, 170), "m4l", "weight");
+   auto df_other = df_mc.Filter("sample_category == \"other\"")
+                      .Histo1D<float>(ROOT::RDF::TH1DModel("other", "m4l", 24, 80, 170), "m4l", "weight");
+
+   // Book the invariant mass histogram for the data
+   auto df_h_mass_data = df_analysis.Filter("reweighting == false")
+                            .Filter("sample_category == \"data\"")
+                            .Define("weight_", []() { return 1; })
+                            .Histo1D<float>(ROOT::RDF::TH1DModel("data", "m4l", 24, 80, 170), "m4l", "weight_");
+
+   // Evaluate the systematic uncertainty
+
+   // The systematic uncertainty in this analysis is the MC scale factor uncertainty that depends on lepton
+   // kinematics such as pT or pseudorapidity.
+   // Muons uncertainties are negligible, as stated in https://atlas.web.cern.ch/Atlas/GROUPS/PHYSICS/PAPERS/MUON
+   // Electrons uncertainties are evaluated based on the plots available in https://doi.org/10.48550/arXiv.1908.0
+   // The uncertainties are linearly interpolated to cover a range of pT values covered by the analysis.
+
+   const std::vector<double> &x = {5.50 * 10e2,  5.52 * 10e2,  12.54 * 10e2, 17.43 * 10e2,
+                                   22.40 * 10e2, 27.48 * 10e2, 30 * 10e2,    10000 * 10e2};
+   const std::vector<double> &y = {0.06628, 0.06395, 0.06396, 0.03372, 0.02441, 0.01403, 0, 0};
```

Comment:
```suggestion
   const std::vector<double> x{5.50e3, 5.52e3, 12.54e3, 17.43e3, 22.40e3, 27.48e3, 30e3, 10000e3};
   const std::vector<double> y{0.06628, 0.06395, 0.06396, 0.03372, 0.02441, 0.01403, 0, 0};
```
Why use this relatively specific C++ feature of initializing a `const&` with an lvalue, instead of just initializing a variable?

Also, you can be more brief by applying the scientific notation directly to the number.